js/locales/bootstrap-datepicker.hu.js

```diff
@@ -4,13 +4,15 @@
  */
 ;(function($){
   $.fn.datepicker.dates['hu'] = {
-		days: ["Vasárnap", "Hétfő", "Kedd", "Szerda", "Csütörtök", "Péntek", "Szombat"],
-		daysShort: ["Vas", "Hét", "Ked", "Sze", "Csü", "Pén", "Szo"],
-		daysMin: ["Va", "Hé", "Ke", "Sz", "Cs", "Pé", "Sz"],
-		months: ["Január", "Február", "Március", "Április", "Május", "Június", "Július", "Augusztus", "Szeptember", "Október", "November", "December"],
-		monthsShort: ["Jan", "Feb", "Már", "Ápr", "Máj", "Jún", "Júl", "Aug", "Sze", "Okt", "Nov", "Dec"],
-		today: "Ma",
+		days: ["vasárnap", "hétfő", "kedd", "szerda", "csütörtök", "péntek", "szombat"],
+		daysShort: ["vas", "hét", "ked", "sze", "csü", "pén", "szo"],
+		daysMin: ["V", "H", "K", "Sze", "Cs", "P", "Szo"],
+		months: ["január", "február", "március", "április", "május", "június", "július", "augusztus", "szeptember", "október", "november", "december"],
+		monthsShort: ["jan", "feb", "már", "ápr", "máj", "jún", "júl", "aug", "sze", "okt", "nov", "dec"],
+		today: "ma",
 		weekStart: 1,
+		clear: "töröl",
+		/* headerFormat: "yyyy. MM", */ /* for month selection */
```

Comment:
Hungarian would need the year+month header to be in the specified 
format, however I could not find in the documentation the proper keyword 
to use for this.

So I just added that commented out line to indicate demand for such a 
setting, and provide the correct value should this setting already exist 
or get implemented in the future.

2015.09.02. 15:37 keltezéssel, Jeroen Thora írta:
>
> In js/locales/bootstrap-datepicker.hu.js 
> <https://github.com/eternicode/bootstrap-datepicker/pull/1553#discussion_r38531650>:
>
> >  		weekStart: 1,
> > +		clear: "töröl",
> > +		/* headerFormat: "yyyy. MM", */ /* for month selection */
>
> why is this line commented out?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/eternicode/bootstrap-datepicker/pull/1553/files#r38531650>.
>